db/views/analytics_clients_v03.sql

```diff
@@ -0,0 +1,49 @@
+SELECT "id",
+  "data_source_id",
+  "PersonalID",
+  "NameDataQuality",
+  "SSNDataQuality",
+  "DOBDataQuality",
+  "AmIndAKNative",
+  "Asian",
+  "BlackAfAmerican",
+  "HispanicLatinaeo",
+  "MidEastNAfrican",
+  "NativeHIPacific",
+  "White",
+  "RaceNone",
+  "AdditionalRaceEthnicity",
+  "Woman",
+  "Man",
+  "NonBinary",
+  "CulturallySpecific",
+  "Transgender",
+  "Questioning",
+  "DifferentIdentity",
+  "GenderNone",
+  "DifferentIdentityText",
+  "VeteranStatus",
+  "YearEnteredService",
+  "YearSeparated",
+  "WorldWarII",
+  "KoreanWar",
+  "VietnamWar",
+  "DesertStorm",
+  "AfghanistanOEF",
+  "IraqOIF",
+  "IraqOND",
+  "OtherTheater",
+  "MilitaryBranch",
+  "DischargeStatus",
+  "DateCreated",
+  "DateUpdated",
+  "UserID",
+  "DateDeleted",
+  "ExportID",
+  "consent_form_id",
+  "housing_release_status",
+  "consent_form_signed_on",
+  "consent_expires_on",
+  "consented_coc_codes"
```

Comment:
@ttoomey do you think we should include these (that was the original request) or just expose the ClientRoiAuthorizations to OP analytics?